lamin_cli/__main__.py

```diff
@@ -328,12 +329,52 @@ def save(filepath: str, key: str, description: str, registry: str):
     if save_from_filepath_cli(filepath, key, description, registry) is not None:
         sys.exit(1)
 
+@main.command()
+@click.option("--path", type=str, default=None, help="The path to the script to run.", required=True)
+@click.option("--app_name", type=str, default=None, help="The name of the app to run the script on.", required=True) # IMPORTANT: Once an image is built it is tied to this app. Artifact key is also tracked here.
+# Super important: Maybe in the future lamin can be empowered by an image registry?... 
+@click.option("--image", type=str, default=None, help="A public image or in the future one that Lamin has access to")
+@click.option("--cpu", type=int, default=4, help="The number of CPUs to use.")
+@click.option("--packages", type=str, default=None, help="A list of packages to install comma seperated") # DEMO ONLY....JUST FOR DEMO, we need like a config..
+@click.option("--gpu", type=str, default=None, help="The type of GPU to use, only compatible with cuda images, reference the tutorials.")   
+def run(path:str, app_name:str, image:str, cpu:int, packages:str, gpu:str):
+    """Run a compute job."""
+    import os
+    import shutil 
+    from lamin_cli.compute.modal import Runner
+    
+    # DEFAULT DIRECTORY TO MOUNT SCRIPTS LOCALLY
+    default_mount_dir = './lamin_scripts' # Should be part of lamin.compute.settings? something like that 
+    
+    # MAKE THE DEFAULT DIRECTORY
+    if not os.path.exists(default_mount_dir):
```

Comment:
Please use `pathlib` everywhere instead of `os.path`. We don't have a single occurance of `os.path` in the code base 😅